db/db_impl/db_impl.cc

```diff
@@ -1496,6 +1496,28 @@ bool DBImpl::SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) {
   }
 }
 
+Status DBImpl::GetFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                   std::string* ts_low) {
+  if (ts_low == nullptr) {
+    return Status::InvalidArgument("Input ts_low is nullptr");
+  }
+  ColumnFamilyData* cfd;
+  if (column_family == nullptr) {
+    cfd = default_cf_handle_->cfd();
+  } else {
+    auto cfh = static_cast_with_check<ColumnFamilyHandleImpl>(column_family);
+    assert(cfh != nullptr);
+    cfd = cfh->cfd();
+  }
+  if (cfd->user_comparator()->timestamp_size() == 0) {
```

Comment:
Nit: assert(cfd) and assert(cfd->user_comparator()) to make linter happy

Reply:
fixed